travis-ci-requirements.txt

```diff
@@ -1,6 +0,0 @@
-numpy
-coverage
-pillow
-git+http://github.com/nucleic/kiwi.git#egg=kiwi
```

Comment:
This file looks like it was unused, but I don't see these anywhere else explicitly. Im assuming they come from enable, and so don't need to be listed as deps

Reply:
not sure where these came from. i didn't bother looking for when this was added to the repo. but you're probably right - they were possible needed for source installs of enable, which are handled by the ci module now.